### PR TITLE
release 3.21.0

### PR DESCRIPTION
**Pull request by RenkuBot**
Release 3.21.0

This PR is auto-generated by [actions/github-script](https://github.com/actions/github-script).

/deploy renku=release-0.50.x

### CODE REVIEW

**Review by ciyer — DISMISSED** [review]
Looks great! Thanks for compiling this release.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,43 @@
...
+
+### Features
+
+- change the url structure for Renku 2.0 projects ([#3026](https://github.com/SwissDataScienceCenter/renku-ui/pull/3026))
+- add an additional top bar for Renku 2.0 ([#3030](https://github.com/SwissDataScienceCenter/renku-ui/pull/3030))
+- ad-hoc feature flag for Renku 2.0 ([#3051](https://github.com/SwissDataScienceCenter/renku-ui/pull/3051))
+- adapt session storage to repository ([#3057](https://github.com/SwissDataScienceCenter/renku-ui/issues/3057), [#3058](https://github.com/SwissDataScienceCenter/renku-ui/pull/3058))
+- add support for entity tags on Renku 2.0 projects ([#3067](https://github.com/SwissDataScienceCenter/renku-ui/pull/3067))
+- Renku 2.0 Sessions ([#3037](https://github.com/SwissDataScienceCenter/renku-ui/pull/3037))
+- Search and Discovery for Renku 2.0 ([#3032](https://github.com/SwissDataScienceCenter/renku-ui/pull/3032))
```
Comment (andre-code) [inline CHANGELOG.md:13]: Do yo agree to rewrite this to something like:
- add sessions for Renku 2.0
- add search for projects and users on Renku 2.0

Comment (leafty) [reply CHANGELOG.md]: Done in 5e6edffd9184340fb43d7c20652a7aab115a55d5.

**Review by andre-code — APPROVED** [review]
LGTM 🚀

**Pull request merged**